content/docs/forestry/migrate.mdx

```diff
@@ -1,17 +1,21 @@
 ---
 title: Migrate from Forestry
-prev: /docs/forestry/overview
-next: /docs/forestry/common-errors
 ---
 
 ## Introduction
 
 TinaCMS can be added to your site locally. In this doc, we'll guide you through the local setup, as well as editing on your production site.
 
 <div style="position:relative;padding-top:56.25%;">
-  <iframe width="560" frameborder="0" allowfullscreen
-      style="position:absolute;top:0;left:0;width:100%;height:100%;" src="https://www.youtube.com/embed/OHY8PyeLK1g" title="Migrating from TinaCMS to Forestry" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share">
-  </iframe>
+  <iframe
```

Comment:
This should be replaced with youtube embed component

Reply:
✅